src/server/game/Entities/Unit/Unit.h

```diff
@@ -334,12 +334,12 @@ enum UnitMods
     UNIT_MOD_STAT_STAMINA,
     UNIT_MOD_STAT_INTELLECT,
     UNIT_MOD_HEALTH,
-    UNIT_MOD_MANA,                                          // UNIT_MOD_MANA..UNIT_MOD_RUNIC_POWER must be in existed order, it's accessed by index values of Powers enum.
+    UNIT_MOD_MANA,                                          // UNIT_MOD_MANA..UNIT_MOD_PAIN must be in existed order, it's accessed by index values of Powers enum.
```

Comment:
Since you have edited the comment on this line already, please improve the English wording like this:
```
// UNIT_MOD_MANA...UNIT_MOD_PAIN must be listed in existing order, it is accessed by index values of Powers enum.
```